roles/nfs/common/tasks/configure.yml

```diff
@@ -135,6 +135,18 @@
   with_items:
   - "{{ scale_protocols.export_ip_pool }}"
 
+- name: configure | Add IPv6 interface
+  shell:
+   cmd: "{{ scale_command_path }}mmces interface add --nic {{ item }}"
+  when: scale_protocols.scale_ces_ipv6_enable and scale_protocols.interface|length > 0
+  with_items:
+  - "{{ scale_protocols.interface }}"
```

Comment:
I ran jenkins , its failing . Add when condition if its variabled defined . this check should done everywhere.